index.html

```diff
@@ -28,18 +29,30 @@
 
         
         <nav>
-            <img class="nav" src="Images/menu-button.png" alt="">
+            <img class="nav" id="myBtn" src="Images/menu-button.png" alt="">
             <ul class="nav-link">
                 <li><a href="">Portfolio</a></li>
                 <li><a href="">About</a></li>
                 <li><a href="">Contact</a></li>
             </ul>
         </nav>
```

Comment:
- [ ] You did a great job 😃 at adding the mobile menu, but I see that there is a line below every link that does not make it like the Figma Design, so please try to remove it to be like the Figma Design 😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/225029595-fa8f7cfe-848a-4cc4-8274-5a8ced8182e9.png)    |    ![image](https://user-images.githubusercontent.com/83360636/225029742-4ec16034-ace9-43b2-8968-10464527559d.png)  |